src/components/ShippingDetails/ShippingDetails.js

```diff
@@ -1,43 +1,39 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import styled from 'styled-components';
 
+import { colorLightGrey } from '../../styles/designTokens';
+import { formatDate } from '../../utils';
 import Shipping from '../Icons/Shipping';
 import Text from '../Text';
-import { colorLightGrey } from '../../styles/designTokens';
+import ArrivalDates from './ArrivalDates';
+import Message from './Message';
+import MessageTitle from './MessageTitle';
+import Section from './Section';
+import Wrapper from './Wrapper';
 
-const Wrapper = styled.div`
-  display: flex;
-  align-items: center;
-`;
+function ShippingDetails() {
+  const arrivalDates = () => {
```

Comment:
getArrivalDates

Reply:
I actually disagree, 'get' prefix here is unnecessary imho.